downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -208,3 +208,35 @@ public function main() {
         string|crypto:PublicKey certFile?;
     |};
     ```
+
+#### `graphql` package
+
+- Add support to GraphQL query complexity analysis, which can be used to enhance the GraphQL service security by mitigating the risk of denial-of-service attacks. With this update, the `graphql:ServiceConfig` record now includes a new field `queryComplexityConfig` to configure the query complexity analysis.
+
+    ```ballerina
```

Comment:
Usually for library modules we seem to be adding just a one or two line description, without including examples. This is a bit of a deviation. https://github.com/ballerina-platform/ballerina-dev-website/blob/release-2201.10.0/downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md#new-features-2

I am +1 for adding examples though.

Reply:
Removed the example to keep the consistency. We have to have a discussion on whether to add the examples here or not.